openapi_tester/clients.py

```diff
@@ -22,14 +22,21 @@ def __init__(
     ) -> None:
         """Initialize ``OpenAPIClient`` instance."""
         super().__init__(*args, **kwargs)
+        #
         self.schema_tester = schema_tester or self._schema_tester_factory()
 
     def request(self, **kwargs) -> Response:  # type: ignore[override]
         """Validate fetched response against given OpenAPI schema."""
         response = super().request(**kwargs)
+        if self._is_successfull_response(response):
```

Comment:
tiny typo here :slightly_smiling_face: 
```suggestion
        if self._is_successful_response(response):
```

Reply:
Done 👍